src/modules/client/components/ClientProfileCard.tsx

```diff
@@ -377,14 +385,13 @@ const ClientProfileCard: React.FC<Props> = ({ client }) => {
               <Skeleton
                 variant='rectangular'
                 sx={{
-                  height: 150,
-                  width: 150,
-                  mr: 1,
+                  height: size,
```

Comment:
This is a small aesthetic change, to make the loader the same size as the image tile, so that the UI doesn't jump around once the image has loaded. I'm guessing it was the original intention but then one got changed w/out the other being updated - but please let me know if there is a reason for it, I'm happy to revert!